### ISSUE TITLE
Add pre-built downloads

### ISSUE THREAD

**Issue by ghost** [mirdaki/ping#21]
Most other open source projects are available in a pre-built form. Many potential users may be turned away by the necessity of having Rust installed. I suggest that pre-built downloads be added to the release section of this github repo.

**Comment by mirdaki** [comment]
Yes! I will do so once the project provides real functionality (right now it just loads some fake messages). I should also update the README to point to the releases so people can easily find the binaries once they're live. Thank you for creating the task!
